packages/next/README.md

```diff
@@ -2628,7 +2628,7 @@ You can use [now dev](https://zeit.co/docs/v2/development/basics) as your local
   ],
   "routes": [
     { "src": "/docs/_next(.*)", "dest": "docs/_next$1" },
-    { "src": "/docs(.*)", "dest": "docs/docs$1" },
+    { "src": "/docs(.*)", "dest": "docs/$1" },
```

Comment:
```suggestion
    { "src": "/docs(.*)", "dest": "docs$1" },
```
Shouldn't we not add the `/` also since it's being captured?

Reply:
Additionally, this is missing `{ continue: true }` so it doesn't actually work in practice 😄.